api/tests/unit/edge_api/identities/test_edge_identity_models.py

```diff
@@ -38,17 +40,10 @@ def test_get_all_feature_states_for_edge_identity_uses_segment_priorities(
     segment_override_p1 = FeatureState.objects.create(
         feature=feature, environment=environment, feature_segment=feature_segment_p1
     )
-    segment_override_p2 = FeatureState.objects.create(
+    FeatureState.objects.create(
         feature=feature, environment=environment, feature_segment=feature_segment_p2
     )
 
-    FeatureStateValue.objects.filter(feature_state=segment_override_p1).update(
-        string_value="p1"
-    )
-    FeatureStateValue.objects.filter(feature_state=segment_override_p2).update(
-        string_value="p2"
-    )
```

Comment:
These seemed superfluous since we're using the feature state's ID for the assertion below.